b2share/modules/communities/errors.py

```diff
@@ -39,3 +39,7 @@ class CommunityDoesNotExistError(Exception):
 class CommunityDeletedError(Exception):
     """Exception raised when a requested community is marked as deleted."""
     pass
+
+class GetAllStartStopError(Exception):
```

Comment:
As said above, [The generic python `ValueError`](https://docs.python.org/3/library/exceptions.html#ValueError) does exactly that.